### ISSUE TITLE
Which model is better for modern French as a pre-training model

### ISSUE THREAD

**Issue by Lier007** [Calamari-OCR/calamari_models#10]

**Comment by ChWick** [comment]
The target language is not relevant since no (strong) language model is learned. Select the model that was trained on fonts that look similar to your target font.

**Comment by Lier007** [comment]
Thank you so much. I didn't make myself clear. Now my situation is like this:
I have 500 French data with labels. I want to label more French data, so I want to fine-tune a model as a start. (This method works well in English with 'uw3-modern-english' as pre-training model ).
So do you have any suggestions, I am very grateful.

**Comment by chreul** [comment]
there are several somewhat suitable models and all have some advantages and disadvantages. personally, i would try uw3 or one of the two french ones. if you want to train a voting ensemble i recommend using a mixed ensemble as a starting point. for example using model 0 from each of the french, uw3, and idiotikon ensembles.

**Comment by Lier007** [comment]
Thank you so much. I have tried that.
I finetuned those related to French, and then ensembled all of them.
Maybe my font is quite different from these, and the final effect is not very good.

**Comment by Lier007** [comment]
Thank you very much, I will close this issue and try something else.
